Terminal.Gui/Views/RadioGroup.cs

```diff
@@ -97,7 +97,7 @@ public RadioGroup ()
                         {
                             SelectedItem = (int)ctx.KeyBinding?.Context!;
 
-                            return !OnAccept();
+                            return OnAccept () != false;
```

Comment:
~~return OnAccept () != false;~~

```csharp
return OnAccept () is not false;
```

Combines the missing null check with a non-overridable type check.

If you want to be clearer, since the tri-state because of null is not all that apparent in a vacuum, you could do this instead:

```csharp
return OnAccept () is true or null;
```

Either way works with the same result.